app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -12,7 +12,10 @@ module Blacklight::BlacklightHelperBehavior
   #
   # @return [String] the application name
   def application_name
-    t('blacklight.application_name')
+    return t('blacklight.application_name') if I18n.locale == :en
+
+    # fallback to :en if no translation exists
+    t('blacklight.application_name', default: t('blacklight.application_name', locale: :en))
```

Comment:
Could we check `I18n.default_locale` (either instead of, or before, checking explicitly for `en`)?